catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -98,5 +102,14 @@ public void init() {
                 .filter(Objects::nonNull)
                 .collect(Collectors.toList()),
         MAPPER::toJson);
+    delete(
+        "/forms/:id",
+        APPLICATION_JSON,
+        (req, res) -> {
+          String id = req.params(":id");
+          catalogFramework.delete(new DeleteRequestImpl(id));
```

Comment:
Do we want to throw a 500 if there are errors?

```
DeleteResponse deleteResponse = catalogFramework.delete(new DeleteRequestImpl(id));
if (deleteResponse.getProcessingErrors() != null && deleteResponse.getProcessingErrors().isEmpty()) {
      res.status(500);
}
```

Reply:
![screen shot 2018-04-03 at 9 30 58 am](https://user-images.githubusercontent.com/7055226/38262324-c18d35d4-3721-11e8-9b89-e0bb64317963.png)

If there is an error, problematic outcome would look like this currently.

Reply:
Setting the response to 500 on error and logging